packages/eslint-plugin/src/rules/no-parameter-properties.ts

```diff
@@ -51,7 +51,6 @@ export default util.createRule<Options, MessageIds>({
                 'public readonly',
               ],
             },
-            minItems: 1,
```

Comment:
Removed this constraint so the default is able to be validated against the schema.

Alternative could be to exclude this from the schema validation list, or have an 'override' schema for testing against the schema.

I don't think this matters for actual usage, and I think it's not harmful: in fact it might be useful in some configs to be able to specify some allowed things by default, and then specify none to be allowed in an override config explicitly to be clear about why it's there.

Reply:
Sorry, I don't follow - why is this `minItems` not good? It's a correct restriction, no?

Reply:
For this PR, the main reason I did it is beacuse it would prefer the default options from validating against the schema itself, and this is a simple way of getting around that.

But also I think semantically it makes sense that it should accept an array of nothing for the exceptions allowed, meaning that no parameter properties are allowed.